### PR TITLE
ci: Update templates based on latest Uno repository

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        - Solution Templates
+
+  - type: dropdown
+    id: ide
+    attributes:
+      label: IDE
+      multiple: true
+      options:
+        - Visual Studio 2022
+        - Visual Studio 2019
```
Comment (Youssef1313) [inline .github/ISSUE_TEMPLATE/bug-report.yml:98]: Probably not relevant anymore?

Comment (jeromelaban) [reply .github/ISSUE_TEMPLATE/bug-report.yml:98]: yes we can remove

**Comment by MartinZikmund** [comment]
@jeromelaban Can you force merge? Seems the PR labeler is failing, but otherwise it should be ok

**Comment by Youssef1313** [comment]
#12 would fix the failure, after it's merged.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    id: how-to-reproduce
+    attributes:
+      label: How to reproduce it (as minimally and precisely as possible)
+      description: |
+                    Please provide a **MINIMAL REPRO PROJECT** and the **STEPS TO REPRODUCE**
+
+                    To create a minimal reproduction project:
+
+                    - Create an Uno app through:
+                    - `dotnet new --install Uno.ProjectTemplates.Dotnet` and `dotnet new unoapp`
```
Comment (Youssef1313) [inline .github/ISSUE_TEMPLATE/bug-report.yml:25]: We shouldn't be mentioning this package.

**Review by Youssef1313 — APPROVED** [review]

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**